src/classes/UTIL_GlobalProxy.cls

```diff
@@ -0,0 +1,227 @@
+    * @description Sends error notifications to the receivers specified in the settings, if error notifications are enabled, with
+    *              all the existing error records that have not been included in previous error notifications.
+    * @param context The context that triggered the error notification.
+    * @return void
+    */
+    global void sendErrorNotifications(ErrorContext context) {
+        ERR_Notifier.sendErrorNotifications(context);
+    }
+    
+    /*******************************************************************************************************
+    * @description Prints a debug message to the debug log, if debugging is enabled or a test is being run.
+    * @param message The message to print.
+    * @return void
+    */
+    global static void debug(string message) {
+    	UTIL_Debug.debug(message);
+    }
+    
+   /*******************************************************************************************************
+    * @description Prints a debug message to the debug log, if debugging is enabled or a test is being run, at the specified
+    *              logging level.
+    * @param message The message to print.
+    * @param level The level to output the message at.
+    * @return void
+    */
+    global static void debug(System.LoggingLevel level, string message) {
+    	UTIL_Debug.debug(level, message);
+    }
+    
+    /*******************************************************************************************************
+    * @description Prints a debug message to the debug log, if debugging enabled or a test is being run, with additional info, 
+                   such as heap, cpu, query, and rows usage.
+    * @param message The message to print.
+    * @return void
+    */
+    global static void debugWithInfo(string message) {
+     	UTIL_Debug.debugWithInfo(message);
+    }
+    
+    /*******************************************************************************************************
+    * @description Prints a debug message to the debug log, if debugging is enabled or a test is being run, at the specified logging
+    *              level, with additional info, such as heap, cpu, query, and rows usage.
+    * @param message The message to print.
+    * @param level The level to output the message at.
+    * @return void
+    */
+    global static void debugWithInfo(System.LoggingLevel level, string message) {
+    	UTIL_Debug.debugWithInfo(level, message);
+    }
+    
+    /*******************************************************************************************************
+    * @description Returns object describe data.
+    * @param objectName The name of the object to lookup. 
+    * @return Schema.DescribeSObjectResult Object that contains methods for describing the SObject.
+    */
+    global static Schema.DescribeSObjectResult getObjectDescribe(String objectName) {
```

Comment:
I'm against exposing these, since they are not specific to NPSP.  But if we do expose them, then I think it should be part of the appropriate class, not in this bucket class.